integration/integrate-with-image-vulnerability-scanners.adoc

```diff
@@ -30,11 +30,20 @@ This enhanced support gives you greater flexibility and choice in managing your
 [discrete]
 == Supported Scanners
 
-You can set up {product-title-short} to obtain image vulnerability data from the following commercial container image vulnerability scanners:
+You can set up {product-title-short} to obtain image vulnerability data from the following commercial container image vulnerability scanners.
+
+Scanners included in {product-title-short}:
+
+* Scanner V4 (Technology Preview): Beginning with {product-title-short} version 4.4, a new scanner is introduced that is built on link:https://github.com/quay/claircore[ClairCore], which also powers the link:https://github.com/quay/clair[Clair] scanner. Scanner V4 supports scanning of language and OS-specific image components. You do not have to create an integration to use this scanner, but you must enable it during or after installation. For version 4.4, if you enable this scanner, you must also enable the StackRox Scanner. For more information about Scanner V4, including links to the installation documentation, see xref:../operating/examine-images-for-vulnerabilities.adoc#about-scanner-v4_examine-images-for-vulnerabilities[About {product-title-short} Scanner V4].
```

Comment:
Just checking if this GitHub link was okayed

Reply:
I don't think ACS has a restriction against linking to github, as far as I know (well, historically they have included github links in the doc content). We don't have a QE group and this link was provided by Engineering.